api/pom.xml

```diff
@@ -87,7 +87,11 @@
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>net.thisptr</groupId>
+            <artifactId>jackson-jq</artifactId>
+            <version>0.0.7</version>
```

Comment:
The specific version should go in the dependencyManagement section of the main pom (it helps keep versions in sync across modules)